graphql-kotlin-federation/build.gradle.kts

```diff
@@ -1,9 +1,11 @@
 description = "Federated GraphQL schema generator"
 
+val kotlinVersion: String by project
 val junitVersion: String by project
 
 dependencies {
     api(project(path = ":graphql-kotlin-schema-generator"))
+    implementation(kotlin("reflect", kotlinVersion))
```

Comment:
I was getting the following errors locally without changing this

```
> Task :graphql-kotlin-federation:compileTestKotlin
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-reflect/1.3.61/kotlin-reflect-1.3.61.jar (version 1.3)
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib/1.4.0/kotlin-stdlib-1.4.0.jar (version 1.4)
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-common/1.4.0/kotlin-stdlib-common-1.4.0.jar (version 1.4)
w: Consider providing an explicit dependency on kotlin-reflect 1.4 to prevent strange errors
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
```